ampd/ampd-release.md

```diff
@@ -20,10 +20,13 @@ This document outlines the step-by-step process for releasing a new version of `
    - **IMPORTANT**: Enable "Dry run" to verify everything works correctly and make a note of the updated tag.
 
 
-### 2. Ensure the Changelog is updated
+### 2. Create release document and ensure the Changelog is updated
```

Comment:
separate this into two independent steps for clarity? create a changelog, and then create the release doc in deployments repo